data/json/monster_special_attacks/monster_deaths.json

```diff
@@ -0,0 +1,568 @@
+[
+  {
+    "id": "necro_boomer_revive",
+    "type": "SPELL",
+    "name": { "str": "Necro Boomer Revive" },
+    "description": "Revives zombies in a large AOE",
+    "valid_targets": [ "ground" ],
+    "flags": [ "SILENT" ],
```

Comment:
```suggestion
    "flags": [ "SILENT" ],
```
Should zombies titled as boomer's explode silently?

Reply:
woops! fixed it. note that it's multiple spells and only one of them has the actual sound.